src/statisticalmodel.jl

```diff
@@ -162,6 +162,15 @@ Return the weights used in the model.
 """
 function weights end
 
+
+"""
+    hasintercept(model::StatisticalModel)
+
+Return a Boolean indicating whether the model has an intercept
```

Comment:
Usually the fact that a Boolean is returned is left implicit:
```suggestion
Indicate whether the model has an intercept.
```